backend/game_server/lib/cell.ex

```diff
@@ -1,27 +1,30 @@
 defmodule BernacleServer.Cell do
-	alias BernacleServer.{Entity,Cell}
-	alias BernacleServer.Helpers.{Physic,Vector}
+	alias BernacleServer.{Entity, Cell}
+	alias BernacleServer.Helpers.{Physic, Vector}
 
 
 	def born(position, velocity) do
-		Entity.new
+		Entity.new()
 			|> Entity.set_attribute(:position, position)
 			|> Entity.set_attribute(:velocity, velocity)
 			|> Entity.set_attribute(:mass, 5)
 			|> Entity.give_ability(:move, Cell)
 			|> Entity.give_ability(:eat, Cell)
-
 	end
 
 	def born(position), do: born(position, %Vector{})
 
 	def born(), do: born(%Vector{}, %Vector{x: 1, y: 1})
 
-	def move(cell, time) do
+	def move(cell = %Entity{}, time) do
 		position = Entity.get_attribute(cell, :position)
 		velocity = Entity.get_attribute(cell, :velocity)
-		new_position = Physic.move(position, velocity, time)
-		Entity.set_attribute(cell, :position, new_position)
+		new_velocity = Physic.calc_current_forces(position, velocity)
```

Comment:
u need take time into account when u calculate how much influence specified force have to volacity